package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ibm-ldap",
-  "version": "1.0.1",
+  "version": "2.0.1",
```

Comment:
Yeah, but my config will also add a tag and make a release, so I prefer to do it from here if you don't mind